examples/OpenTracing.Examples/LateSpanFinish/LateSpanFinishTest.cs

```diff
@@ -46,15 +46,15 @@ private Task SubmitTasks()
             {
                 using (IScope childScope1 = _tracer.BuildSpan("task1").StartActive(finishSpanOnDispose:true))
                 {
-                    await Task.Delay(55);
+                    await Task.Delay(100);
```

Comment:
Are these changes related to the new binary propagator?

Reply:
No, they're adjusting test timings because we keep having PR's fail. Admittedly it's not great in this PR but I didn't want to make a separate PR and rebase just to get it.